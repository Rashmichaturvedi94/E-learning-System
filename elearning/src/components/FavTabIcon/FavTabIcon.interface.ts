import { TabBarIconProps } from 'components/TabBarIcon/TabBarIcon.interface';

export type FavTabIconProps = Omit<TabBarIconProps, 'Icon'>;
