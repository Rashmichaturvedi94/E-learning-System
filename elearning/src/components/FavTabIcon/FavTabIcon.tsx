import React, { FC } from 'react';
import { TabBarIcon } from 'components/TabBarIcon';
import { FavIcon } from 'components/Icon';
import { FavTabIconProps } from './FavTabIcon.interface';

export const FavTabIcon: FC<FavTabIconProps> = ({ focused, ...iconProps }) => (
  <TabBarIcon
    focused={focused}
    Icon={FavIcon}
    width={32}
    height={32}
    {...iconProps}
  />
);
