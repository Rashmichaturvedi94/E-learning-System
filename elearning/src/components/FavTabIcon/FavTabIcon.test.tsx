import React from 'react';
import { render } from '@testing-library/react-native';
import { FavTabIcon } from './FavTabIcon';

describe('components/FavTabIcon', () => {
  it('should render', () => {
    const component = render(<FavTabIcon />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
