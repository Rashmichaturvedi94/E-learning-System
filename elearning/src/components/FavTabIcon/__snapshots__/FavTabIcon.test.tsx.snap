// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/FavTabIcon should render 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "opacity": 0.6,
      },
    ]
  }
>
  <SvgMock
    height={32}
    width={32}
  />
</View>
`;
